packages/postman-to-openapi/src/convert.ts

```diff
@@ -0,0 +1,771 @@
+import type { OpenAPIV3 } from '@scalar/openapi-types'
+import { parse } from 'jsonc-parser'
+import camelCase from 'lodash.camelcase'
+
+import type {
+  Options,
+  PostmanUrl,
+  PostmanVariable,
+  SecurityDefinition,
+} from './types'
+
+function parseMdTable(): Record<string, any> {
+  return {}
+}
+
+function replacePostmanVariables(collectionFile: any): any {
+  return collectionFile
+}
+/**
+ * Converts a Postman collection to an OpenAPI specification.
+ *
+ * @param {string | Record<string, any>} collectionContent - The Postman collection content, either as a JSON string or an object.
+ * @param {Options} [options={}] - Configuration options for the conversion process.
+ * @returns {Promise<string>} A promise that resolves to the OpenAPI specification as a JSON string.
```

Comment:
We don't need JSDocs for anything as we can directly annotate parameters using typescript.